pyuvdata/utils.py

```diff
@@ -791,24 +791,25 @@ def antnums_to_baseline(ant1, ant2, Nants_telescope, attempt256=False):
         number of antennas
     attempt256 : bool
         Option to try to use the older 256 standard used in
-        many uvfits files (will use 2048 standard if there are more
-        than 256 antennas). Default is False.
+        many uvfits files. If there are more than 256 antennas, the 2048
+        standard will be used unless there are more than 2048 antennas.
+        In that case, the 2147483648 standard will be used. Default is False.
 
     Returns
     -------
     int or array of int
         baseline number corresponding to the two antenna numbers.
 
     """
-    if Nants_telescope is not None and Nants_telescope > 2048:
+    if Nants_telescope is not None and Nants_telescope > 2147483648:
```

Comment:
hmm, do we need this test given that we have the next one (for antenna number)? Maybe we do in case that we are only passing some of the antennas. But if that's what we are worried about then I think we should update the code below to use the largest standard if Nants>2048 even if the antenna numbers that are passed are smaller than the limit.

Reply:
I think that makes sense. I've updated the code to reflect we are checking both Nants>2048 and antenna numbers > 2047, and updated the docstring as well.